scripts/lustre

```diff
@@ -70,8 +70,8 @@ lustre_monitor() {
         if [ $mntdev = $OCF_RESKEY_dataset ] ; then
             ocf_log debug "$OCF_RESKEY_dataset found in $mntdev_path"
             return $OCF_SUCCESS
+        else ocf_log debug "$OCF_RESKEY_dataset does not match $mntdev"
```

Comment:
Note that the other message also appeared only if mntdev did not match.  Because if mntdev did match, we followed the "return" and the code afterwards, including the "ocf_log info", was not executed.